test/metabase/test_runner.clj

```diff
@@ -84,7 +84,7 @@
    "test_resources"])
 
 (defn- default-options []
-  {:namespace-pattern   #"^metabase.*test$"
+  {:namespace-pattern   #"^metabase.*"
```

Comment:
Also unrelated, we have a few random tests in places like `metabase.test-util` and our test runner shouldn't skip these tests just because the namespace doesn't end in `test`. Look for tests in all `^metabase` namespaces.